app/controllers/games_controller.rb

```diff
@@ -0,0 +1,20 @@
+class GamesController < ApplicationController
```

Comment:
@erinmorrissey I actually already created the update method in the UpdateAction branch... I guess we have been working on the same thing... What were you working on? I was doing the "add player to a game"...